jbmc/regression/jbmc-strings/toCodePoint/Test.java

```diff
@@ -0,0 +1,10 @@
+public class Test {
+
+  public void test(char a, char b) {
+    assert Character.toCodePoint((char)0xD800, (char)0xDC00) == 0x10000;
+    assert Character.toCodePoint((char)0xDBFF, (char)0xDFFF) == 0x10ffff;
+    assert Character.toCodePoint((char)0xD800, (char)0xDFFF) == 0x103ff;
+    assert Character.toCodePoint((char)0xDBFF, (char)0xDC00) == 0x10fc00;
+    assert Character.toCodePoint(a, b) == 0x10000;
```

Comment:
To make the test better maybe assert `!= 0x10000` and check in the trace produced by jbmc that it found the correct values for `a` and `b` to falsify the assertion.